multicluster/cmd/multicluster-controller/memberclusterannounce_webhook.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
```

Comment:
`multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"` need to be in a separate import section, please run `make golangci` to avoid this kind of issue.

Reply:
Yes ive done that now. Sorry I moved this commit to a different PR now and have fixed it there already.